app/javascript/flavours/polyam/components/status_reactions.tsx

```diff
@@ -86,16 +87,28 @@ const Reaction: FC<{
     ? reactionName
     : `:${reactionName}:`;
 
+  const extraEmoji: ApiCustomEmojiJSON[] = [
+    {
+      shortcode: reactionName,
+      url: reaction.get('url'),
+      static_url: reaction.get('static_url'),
+      visible_in_picker: true,
+    },
+  ];
+
```

Comment:
This is a bit of a hack.
The reaction object can't be used directly because the keys do not match.
The visible key has to be set to satisfy the type but shouldn't be used in the EmojiHTML component.

Long term it might be worth it to use a serialized emoji attribute in reactions.
This would also make it more consistent with other responses containing custom emojis.